src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -67,6 +109,33 @@ that a given scan was collected with the intended coil elements selected
 
 ### In-Plane Spatial Encoding
 
+{{ MACROS___make_metadata_table(
+   {
+      "NumberShots": "RECOMMENDED",
+      "ParallelReductionFactorInPlane": "RECOMMENDED",
+      "ParallelAcquisitionTechnique": "RECOMMENDED",
+      "PartialFourier": "RECOMMENDED",
+      "PartialFourierDirection": "RECOMMENDED",
+      "PhaseEncodingDirection": (
+         "RECOMMENDED",
+         "This parameter is REQUIRED if corresponding fieldmap data is present "
+         "or when using multiple runs with different phase encoding directions "
+         "(which can be later used for field inhomogeneity correction).",
+      ),
+      "EffectiveEchoSpacing": (
+         "RECOMMENDED",
+         "This parameter is REQUIRED if corresponding fieldmap data is present.",
+      ),
+      "TotalReadoutTime": (
+         "RECOMMENDED",
+         "This parameter is REQUIRED if corresponding 'field/distortion' maps "
+         "acquired with opposing phase encoding directions are present "
+         "(see 8.9.4).",
```

Comment:
Add superscript and also finally update this out-of-date reference to a section number.

```suggestion
         "<sup>3</sup> This parameter is REQUIRED if corresponding 'field/distortion' maps "
         "acquired with opposing phase encoding directions are present "
         "(see [Case 4: Multiple phase encoded "
         "directions](#case-4-multiple-phase-encoded-directions-pepolar)).",
```